src/Magento/FunctionalTestingFramework/Test/Util/TestObjectExtractor.php

```diff
@@ -11,6 +11,7 @@
 use Magento\FunctionalTestingFramework\Test\Objects\ActionObject;
 use Magento\FunctionalTestingFramework\Test\Objects\TestObject;
 use Magento\FunctionalTestingFramework\Util\ModulePathExtractor;
+use Magento\FunctionalTestingFramework\Util\ModuleResolver;
```

Comment:
Is it necessary to add this `use` here?

Reply:
Code has changed and it's not needed any more. Removed.